modmesh/view.py

```diff
@@ -55,6 +57,29 @@
 except ImportError:
     pass
 
+# Try to find the PUI in thirdparty, if failed to find PUI
+# modmesh will raise ImportError and terminate itself.
+filename = os.path.join('thirdparty', 'PUI')
+path = os.getcwd()
+try:
+    while True:
+        if os.path.exists(os.path.join(path, filename)):
+            break
+        if path == os.path.dirname(path):
+            path = None
+            break
+        else:
+            path = os.path.dirname(path)
+    if path is None:
+        raise ImportError
+except ImportError:
+    sys.stderr.write('Can not find PUI in your environment.\n')
+    sys.stderr.write('Please run git submodule update --init\n')
+    sys.exit(0)
```

Comment:
If user forget to initialize the submodule it will raise a `ImportError` and terminate modmesh, also the error message will remind user to initialize the submodule to install the PUI.